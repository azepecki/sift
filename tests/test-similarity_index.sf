def int main() {
    print(get_jaro("THOUGHT",    "THROW"));
    print(get_jaro("ANALYZE",     "PARALYZE"));
    print(get_jaro("JELLYFISH", "SMELLYFISH"));
    return 0;
}
