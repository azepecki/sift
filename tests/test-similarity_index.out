0.676190
0.869048
0.896296
